src/cobra/core/reaction.py

```diff
@@ -224,8 +259,8 @@ def update_variable_bounds(self):
             )
 
     @property
-    def lower_bound(self):
-        """Get or set the lower bound
+    def lower_bound(self) -> Union[int, float]:
```

Comment:
As earlier, please make the return type as `float` and also add a Returns section to the docstring. This method only gets the value, so I believe you can move the sets directive in the docstring to the setter method.